taskrc/taskrc.go

```diff
@@ -1,6 +1,62 @@
 package taskrc
 
+import (
+	"os"
+	"slices"
+
+	"github.com/go-task/task/v3/internal/fsext"
+	"github.com/go-task/task/v3/taskrc/ast"
+)
+
 var defaultTaskRCs = []string{
 	".taskrc.yml",
 	".taskrc.yaml",
 }
+
+// GetConfig loads and merges local and global Task configuration files
+func GetConfig(dir string) (*ast.TaskRC, error) {
+	var config *ast.TaskRC
+	reader := NewReader()
+
+	// Read the XDG config file
+	if xdgConfigHome := os.Getenv("XDG_CONFIG_HOME"); xdgConfigHome != "" {
+		xdgConfigNode, err := NewNode("", xdgConfigHome)
```

Comment:
Middle of the night thought to self. Fix this to be a task subdirectory...

Reply:
I totally agree!